data/human/far north jobs.txt

```diff
@@ -90,6 +90,7 @@ mission "Navy resupply [0]"
 	to offer
 		random < 2
 	source
+		government "Republic" "Syndicate"
```

Comment:
I'd add a distance filter just in case, as well. Sorry for leaving that out and getting it wrong the first time : )